python/dlisio/plumbing/computation.py

```diff
@@ -72,6 +73,8 @@ def __init__(self, obj = None, name = None):
         #: The immediate source of the Computation
         self.source = []
 
+        self._values = np.empty(0)
```

Comment:
I don't like this solution much, but if you don't want to introduce feature attributes, I guess this is the best option.
Reloading the load method and removing values from stash I like way less. Changing load method in baseclass might not be for the best too.